Library/Formula/eg.rb

```diff
@@ -0,0 +1,28 @@
+class Eg < Formula
+  homepage "https://github.com/srsudar/eg"
```

Comment:
I remember the official stance used to be that we didn't want descriptions because the package can easily change, rendering the description obsolete. I'd like to know what changed your mind, if you don't, er… mind.